frontend/device/wakeupmgr.lua

```diff
@@ -240,11 +240,9 @@ end
 --[[--
 Get RTC wakealarm from system.
 
-Simple wrapper for @{ffi.rtc.getWakeupAlarmSys}.
+Simple alias for @{ffi.rtc.getWakeupAlarmSys}.
 --]]
-function WakeupMgr:getWakeupAlarmSys()
-    return RTC:getWakeupAlarmSys()
-end
+WakeupMgr.getWakeupAlarmSys = RTC.getWakeupAlarmSys
```

Comment:
That won't work because RTC is not a module-local, IIRC (it can be overloaded by the constructor).

Reply:
I see a problem: Calling WakeupMgr:getWakeupAlarmSys() will call RTC.getWakeupAlarmSys(self), but I also see that self is not used in RTC.getWakeupAlarmSys (in `ffi/rtc.lua`). 

So it is OK but not ultra clean to use `WakeupMgr.getWakeupAlarmSys = RTC.getWakeupAlarmSys`. But if we want to be ultra  clean we should also rewrite RTC:getWakeupAlarmSys() to RTC.getWakupAlarmSys()

Is that the problem you are adressing here or are you maybe confusing somthing like https://github.com/koreader/koreader/blob/60a16c7e00ac6ac024ef32b702839f95c070d3a2/frontend/device/kindle/powerd.lua#L255 with this. As I understand (although I am not familiar with the code) WakeupMgr has a member called rtc (lowercase). This rtc can be set by a constructor. But the line here is using RTC (uppercase), which is a `local RTC = require("ffi/rtc"))` (and can't be overloaded by anything). ???
(It has been my original intention: Use ` grep -RPzo "\nfunction.*\(\)\n .*\(\)\nend\n" frontend/` to find wrappers and replace them. But it was obvious that `self.something` can not be optimized, so only a few methods remained.)

Reply:
I was thinking of the `rtc = RTC` in the constructor, but you raises an interesting concern, too, so, yeah, i'd forget about this one ;).

(LuaJIT is *very* good at tail-call optimizations anyway).

Reply:
So I will leave this change in, and I am looking forward to the solution of the "interesting concern".

Reply:
I'd revert it ;).